### DISCUSSION TITLE
Removing download button / _sources folder

### DISCUSSION THREAD

**Discussion by bknaepen** [executablebooks/meta#170] [Q&A]
Hi all,

Is there a setting to remove the download button (for .ipynb, .md, .pdf) from the html output?

Also, I make the notebooks available outside of the generated book. Is there a setting to avoid having a _source folder included in the _build directory ?

Cheers,
Bernard.

**Comment by bknaepen** [comment]
Follow up:

to remove the download button, I added this my custom css file:

[aria-label="Download this page"] {
    display: none;
}

and it works...

**Comment by choldgraf** [comment]
Currently I think this is automatically added, but it'd be better if we made this a configurable option. I think the place to do this would be in the `sphinx-book-theme`: https://sphinx-book-theme.readthedocs.io/en/latest/

wanna open up an issue in that repository so that we can add this?

**Reply by bknaepen** [reply]
Sure, I'll do 👍
